chat_server/server_utils/auth.py

```diff
@@ -304,3 +312,16 @@ async def wrapper(request: Request, *args, **kwargs):
         return outer(func)
     else:
         return outer
+
+
+def _is_system_token(session: str) -> bool:
+    return system_token and session == system_token
+
+
+def is_authorized_for_user_id(current_user: CurrentUserModel, user_id: str) -> bool:
```

Comment:
Please add a docstring; it's not clear to me what is being compared here.

Reply:
> Please add a docstring; it's not clear to me what is being compared here.

done